src/__tests__/TopNav.test.tsx

```diff
@@ -1,65 +1,108 @@
+/**
+ * @jest-environment jsdom
+ */
 import React from "react";
-import { shallow } from "enzyme";
+import { configure, mount, shallow } from "enzyme";
 import TopNav from "../components/TopNav";
+import { IconButton } from "@mui/material";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 
 // TODOs: need to figure out the wrapper type before using the following beforeEach code block
 // let wrapper;
 // beforeEach(()=>{
 // 	wrapper = shallow(<TopNav/>);
 // })
 
+configure({ adapter: new Adapter() });
+
 // test component rendering
 describe("<TopNav /> rendering", () => {
     const wrapper = shallow(<TopNav />);
     // test if the TopNav component renders
     it("should be able to render <TopNav />", () => {
         wrapper;
     });
-    // test if there are 9 Link elements
-    it("should render 9 Links", () => {
-        expect(wrapper.find("Link")).toHaveLength(9);
+    // test if there is Typography elements
+    it("should render 3 Typography", () => {
+        expect(wrapper.find("Typography"));
```

Comment:
We already spoke about using expect + matchers so I won't mention all the tests where this needs to change. The Jest matchers can be found [here](https://jestjs.io/docs/using-matchers).